.gitignore

```diff
@@ -143,3 +143,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Mac generated files
+.DS_Store
```

Comment:
I'm personally opposed to adding this, but it's borderline. I usually keep a project's `.gitignore` restricted to project-specific things (i.e., programming language) and leave user-specific things (i.e., operating system, text editor) up to global `~/.gitignore` files.